manifests/install/charts/as-a-second-scheduler/README.md

```diff
@@ -26,7 +26,11 @@ Quick start instructions for the setup and configuration of as-a-second-schedule
 ```bash
 $ git clone git@github.com:kubernetes-sigs/scheduler-plugins.git
 $ cd scheduler-plugins/manifests/install/charts
-$ helm install scheduler-plugins as-a-second-scheduler/
+# Helm will install all components in your corrent-context namespace.
+$ helm install scheduler-plugins as-a-second-scheduler/ 
+# You can use `--namespace` parameter to specify the namespace you want to deploy. 
+# You can also use `--create-namespace` parameter to create it automatically if the specified namespace does not exist.
+$ helm install scheduler-plugins as-a-second-scheduler/ --create-namespace --namespace scheduler-plugins
```

Comment:
I would prefer we provide the `scheduler-plugins` ns by default and provide some guidance to auto-create the ns.

it seems not worth mentioning the context namespace.

```suggestion
helm install scheduler-plugins as-a-second-scheduler/ --namespace scheduler-plugins
    ```

By specifying `--create-namespace` parameter, You can also create the namespace if it does not exist
```

ps, I have to indent the quote in the suggestions

Reply:
WDYT @Huang-Wei

Reply:
sounds good. I will mark this as an incompatible change in the release notes, and here let's just simply suggest:

```
helm install scheduler-plugins as-a-second-scheduler/ --namespace scheduler-plugins --create-namespace
```

to install the Helm chart.